.eslintrc.yaml

```diff
@@ -0,0 +1,178 @@
+env:
+  browser: true
+  node: true
+  es6: true
+plugins:
+    - react
+extends:
+    - eslint:recommended
+    - plugin:react/recommended
```

Comment:
React is not part of the API-Server. Please delete the plugin to prevent false positives etc.